code/__HELPERS/icons.dm

```diff
@@ -872,6 +872,52 @@ The _flatIcons list is a cache for generated icon files.
 	del(atom_icon)
 	return text_image
 
+// Pick a random animal instead of the icon, and use that instead
+/proc/getRandomAnimalImage(atom/A)
+	var/list/icons = list()
+	var/icon/selected
+
+	icons += new/icon('icons/mob/animal.dmi', "cow")

```

Comment:
You could probably loop through all the simple animal subtypes and check for gold_core_spawnable == 2, which determines 'friendly' animals so that you don't end up with wraiths and faithless and whatever other nasty things everywhere.